conf/retention.go

```diff
@@ -24,7 +24,7 @@ type Retentions struct {
 
 // Sub returns a "subslice" of Retentions starting at the given pos.
 func (r Retentions) Sub(pos int) Retentions {
-	origSplit := strings.Split(r.Orig, ":")
+	origSplit := strings.Split(r.Orig, ",")
```

Comment:
how is it possible that this was never noticed if the schema parsing was broken?

Reply:
parsing isn't broken.
only generation of the "orig" string when cutting up a set of retentions into a smaller set.
the Orig field is used for keeping retentions human readable
(it was introduced in #1508 )